src/constants.js

```diff
@@ -156,218 +138,178 @@ export const SAUCE_CONNECT_CLI_PARAMS = [
     description: 'Specify the Sauce Connect version you want to use.',
     default: DEFAULT_SAUCE_CONNECT_VERSION,
   },
-  {
-    /**
-     * Sauce Connect parameter
-     */
-    alias: 'a',
-    name: 'auth',
-    description:
-      'Perform basic authentication when an URL on <host:port> asks for a username and password.',
-  },
-  {
-    name: 'cainfo',
-    description:
-      'CA certificate bundle to use for verifying REST connections. (default "/usr/local/etc/openssl/cert.pem")',
-  },
-  {
-    name: 'capath',
-    description:
-      'Directory of CA certs to use for verifying REST connections. (default "/etc/ssl/certs")',
-    deprecated: true,
-  },
   {
     alias: 'c',
     name: 'config-file',
     description: 'Path to YAML config file.',
   },
   {
-    alias: 'D',
-    name: 'direct-domains',
+    alias: 'i',
```

Comment:
Maintaining such a list requires releasing a new version of node-saucelabs whenever a new parameter is added or modified in the SC client. I wonder if there is a way to avoid it.